exercises/G-conditionals-2/exercise-4.js

```diff
@@ -8,14 +8,20 @@
   Hint: Google how to check if a string contains a word
 */
 
-function containsCode(sentence) {}
+function containsCode(sentence) {
+  if (sentence.includes("code")) {
+    return true;
+  } else {
+    return false;
+  }
+}
```

Comment:
This works fine. You could also do it in 1 line of code `return(any boolean expression)` will return true or false. But what you've done is fine, whichever you prefer.